src/languageservice/jsonSchema.ts

```diff
@@ -90,3 +90,7 @@ export interface JSONSchema {
 export interface JSONSchemaMap {
   [name: string]: JSONSchemaRef;
 }
+
+export function isJSONSchema(schema: JSONSchemaRef): schema is JSONSchema {
+  return !(schema instanceof Boolean);
```

Comment:
I'm not sure this will work as expected. It might, I have not tested it. Capital B Boolean is the class Boolean. 
The var `schema` has a type of `JSONSchemaRef`, can it be a Boolean. Might need to make `schema` a union type. 
If it is a simple boolean then using `typeof` is better.

Since you are asking for the NOT of `instanceof`, it might always return the wrong answer. This needs thorough testing.